config.js

```diff
@@ -0,0 +1,3 @@
+var config = {
```

Comment:
Is this file needed? I noticed that the API key is already inside chart.js

Reply:
This is not needed for now as I searched online that it's not necessary to protect Google map api key.
While I am not sure if we will need it in the future. 
Shall I delete it for now?

Reply:
Yup go ahead and delete it.

Reply:
Done!